mleap-spark/src/main/scala/org/apache/spark/ml/bundle/tree/decision/SparkNodeWrapper.scala

```diff
@@ -43,7 +43,7 @@ object SparkNodeWrapper extends NodeWrapper[tree.Node] {
 
   override def leaf(node: LeafNode, withImpurities: Boolean): tree.Node = {
     val calc: ImpurityCalculator = if(withImpurities) {
-      ImpurityCalculator.getCalculator("gini", node.values.toArray)
+      ImpurityCalculator.getCalculator("gini", node.values.toArray, rawCount = -1L)
```

Comment:
Note: This `rawCount` argument is added in spark 3.0.0. Though, in mleap supporting this argument require a big change. We need to modify tree node proto definition and related serializer in mleap.
So here I simply set it to be `-1` to represent this property is unsupported in mleap. This `rawCount` in spark is just an extra statistic property for tree node and won't be used in model prediction. So leave it empty (-1) will be safe.
We could do the work of completely supporting `rawCount` for mleap  in separate PR